examples/flask_app/web.py

```diff
@@ -15,8 +22,8 @@
 def simple_checkout():
     snap = Snap(
         is_production=False,
-        server_key='SB-Mid-server-GwUP_WGbJPXsDzsNEBRs8IYA',
-        client_key='SB-Mid-client-61XuGAwQ8Bj8LxSS',
+        server_key=SERVER_KEY,
+        client_key=CLIENT_KEY
```

Comment:
It's ok, don't remove the API key. For example web server project folder like this, it is intended it can be run with just 1 single command without modifying the code, for simplicity of demo purpose. If you want to add instruction to change API key, add it in readme of this example. Add it as optional/recommended step.